.github/CODEOWNERS

```diff
@@ -29,3 +48,84 @@ cmake/**/*Windows*    @compnerd
+/include/swift/AST/*Generic*      @hborla @slavapestov
+/include/swift/AST/*Protocol*     @hborla @slavapestov
+/include/swift/AST/*Requirement*  @hborla @slavapestov
+/include/swift/AST/*Substitution* @slavapestov
+/lib/AST/                         @hborla @slavapestov @xedin
+/lib/AST/*Conformance*            @slavapestov
+/lib/AST/Evaluator*               @CodaFi @slavapestov
+/lib/AST/*Generic*                @hborla @slavapestov
+/lib/AST/*Requirement*            @hborla @slavapestov
+/lib/AST/RequirementMachine/      @slavapestov
+/lib/AST/*Substitution            @slavapestov
+/unittests/AST/                   @hborla @slavapestov @xedin
+/unittests/AST/*Evaluator*        @CodaFi @slavapestov
+
+# Sema
+/include/swift/Sema/                     @hborla @slavapestov @xedin
+/include/swift/Sema/Constraint*          @hborla @xedin
+/include/swift/Sema/CS*                  @hborla @xedin
+/lib/Sema/                               @hborla @slavapestov @xedin
+/lib/Sema/Constraint*                    @hborla @xedin
+/lib/Sema/CS*                            @hborla @xedin
+/lib/Sema/CodeSynthesisDistributed*      @hborla @ktoso
+/lib/Sema/DerivedConformance*            @slavapestov
+/lib/Sema/DerivedConformanceDistributed* @ktoso @slavapestov
+/lib/Sema/TypeCheckDistributed*          @hborla @ktoso @xedin
+/lib/Sema/TypeCheckType*                 @AnthonyLatsis @hborla @slavapestov @xedin
+/lib/Sema/TypeCheckProtocol*             @AnthonyLatsis @hborla @slavapestov
+/test/Constraints/                       @hborla @xedin
+/test/decl/                              @hborla @slavapestov
+/test/decl/protocol/                     @AnthonyLatsis @hborla @slavapestov
+# FIXME: This file could have a dedicated directory.
+/test/decl/protocol/special/DistributedActor.swift  @ktoso
+# FIXME: Should there be a 'Sema' folder under 'Distributed'? Or perhaps it
+# should be the other way around, i.e 'Sema/Distributed', 'SILGen/Distributed',
+# etc.?
+/test/Distributed/*                      @ktoso
+# FIXME: Is there a better way to structure this high-level 'Inputs' dir.? We
+# want only the Sema bits here.
+/test/Distributed/Inputs/                @ktoso
+/test/expr/                              @hborla @slavapestov @xedin
+/test/Generics/                          @hborla @slavapestov
+/test/Sema/                              @hborla @slavapestov @xedin
+/test/stmt/                              @hborla @xedin
+/test/type/                              @hborla @slavapestov @xedin
+/validation-test/Sema/                   @hborla @slavapestov @xedin
+/unittests/Sema/                         @hborla @xedin
+
+# SIL/SILGen
+# TODO: /include/swift/SIL/
+# TODO: /lib/SIL/
+# TODO: /lib/SILGen/
+/lib/SILGen/*Distributed* @ktoso
+/test/Distributed/SIL/    @ktoso
+# TODO: /*test/SIL/
+# TODO: /*test/SILGen/
+# FIXME: This file should go inside either 'test/Distributed/SIL' or
+# 'test/SILGen/Distributed'.
+/test/SILGen/distributed_thunk.swift @ktoso
```

Comment:
@ktoso Could you please move this file? It seems like SILGen/Distributed/ is a good spot even if it's just one file (for now).

Reply:
Sure, I'll move it 👍

Reply:
Done in: https://github.com/apple/swift/pull/62357 it'll be `test/Distributed/SIL/distributed_thunk.swift`

Reply:
Thank you!